src/app.js

```diff
@@ -25,11 +37,30 @@ const quoteData = [
   },
 ];
 
+
 $(document).ready(function() {
   const quotes = new QuoteList(quoteData);
+  const orders = new OrderList();
   const simulator = new Simulator({
     quotes: quotes,
   });
 
+  const quoteListView = new QuoteListView({
+    model: quotes,
+    template: _.template($('#quote-template').html()),
+    tradesTemplate: _.template($('#trade-template').html()),
+    el: 'main'
+  });
+
+  const orderListView = new OrderListView({
+    model: orders,
```

Comment:
I would suggest passing `quotes` in to the `OrderListView` and then the view can listen for changes in quotes and check to see if it's time to execute an order.